openwisp_users/tests/test_models.py

```diff
@@ -18,6 +19,23 @@ def test_create_superuser_email_empty(self):
                                              email='')
         self.assertEqual(user.emailaddress_set.count(), 0)
 
+    def test_unique_email_validation(self):
+        self._create_user(username='user1', email='same@gmail.com')
+        with self.assertRaises(ValidationError):
+            options = {
+                'username': 'user2',
+                'email': 'same@gmail.com',
+                'password': 'pass1'
+            }
+            u = User(**options)
+            u.full_clean()
+            u.save()
+
+    def test_create_user_without_email(self):
+        User.objects.create(username='testuser', password='test1', email='')
```

Comment:
do not use `create()`, because it does not run the validation, instantiate the user and then validate it with `full_clean()`